cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/StatefulSetOperator.java

```diff
@@ -105,8 +106,8 @@ public StatefulSetOperator(Vertx vertx, KubernetesClient client, long operationT
             }).compose(ignore -> {
                 Future del = podOperations.waitFor(namespace, name, pollingIntervalMs, timeoutMs, (ignore1, ignore2) -> {
                     // predicate - changed generation means pod has been updated
-                    int newGen = getPodGeneration(podOperations.get(namespace, podName));
```

Comment:
Is `getPodGeneration()` still used somewhere else? If not then you should remove it.

Reply:
It's still used in `isPodUpToDate()`. Do you think i should refactor `isPodUpToDate` to use UID as well?

Reply:
I was just checking it *was* still used. Don't change `isPodUpToDate()`.